DesktopEdge/Views/ItemRenderers/IdentityItem.xaml.cs

```diff
@@ -223,9 +223,9 @@ public void RefreshUI() {
                 ServiceCountArea.Visibility = Visibility.Collapsed;
             }
 
-            IdName.Content = _identity.Name;
-            IdUrl.Content = _identity.ControllerUrl;
-            if (_identity.ContollerVersion != null && _identity.ContollerVersion.Length > 0) IdUrl.Content = _identity.ControllerUrl + " at " + _identity.ContollerVersion;
+            IdName.Text = _identity.Name;
```

Comment:
Referring to the changed required for ellipses assignment, the code was also required to have an update.

https://github.com/openziti/desktop-edge-win/pull/756/commits/9e595faa0483521a7d65dce790db29a1b798c255#r1847156347